packages/stream-model-handler/src/__tests__/model-handler.test.ts

```diff
@@ -805,7 +810,8 @@ describe('ModelHandler', () => {
     const state$ = TestUtils.runningState(state)
     const doc = new Model(state$, context)
     const rawCommit = doc._makeRawCommit(FINAL_CONTENT)
-    rawCommit.header = { controllers: [did.id, did.id] }
+    const newDid = 'did:3:k2t6wyfsu4pg0t2n4j8ms3s33xsgqjhtto04mvq8w5a2v5xo48idyz38l7zzzz'
+    rawCommit.header = { controller: newDid }
```

Comment:
I thought we said not to change the commit headers?

Reply:
left over from old version.  Doesn't really matter to the test as it will error if the header is set at all.  Anyway fixed.